cmd/remote_enforcer.go

```diff
@@ -248,10 +296,15 @@ func (s *Server) Supervise(req rpcwrapper.Request, resp *rpcwrapper.Response) er
 		nil)
 
 	runtime := policy.NewPURuntimeWithDefaults()
+
 	puInfo := policy.PUInfoFromPolicyAndRuntime(payload.ContextID, pupolicy, runtime)
-	puInfo.Runtime.SetPid(os.Getpid())
-	log.WithFields(log.Fields{"package": "remote_enforcer",
-		"method": "Supervise",
+
+	// TODO - Set PID to 1
+	puInfo.Runtime.SetPid(1)
```

Comment:
We changed this from os.Getpid() to 1. This cant be correct. If we dont need this, it should be cleaned up/not called.